lib/karafka.rb

```diff
@@ -11,9 +11,12 @@
   require_all
   dry-configurable
   dry-validation
+  dry/events/publisher
   dry/inflector
   dry/monitor/notifications
-  active_support/callbacks
+  active_support/core_ext/class/attribute
+  active_support/core_ext/hash/except
```

Comment:
do we need this? I believe I removed it recently (guess in 1.2.2). Could you please verify? We aim to remoe AS at all.

Reply:
Still needed:

    karafka/lib/karafka/base_responder.rb:63:in `<class:BaseResponder>': undefined method `class_attribute' for Karafka::BaseResponder:Class (NoMethodError)

    karafka/lib/karafka/setup/configurators/water_drop.rb:16:in `block in setup': undefined method `except' for #<Hash:0x00007fce633972c0> (NoMethodError)

We could remove/provide implementations for those but probably this PR isn't the best place to do it.

Reply:
You're absolutely right :) Thank you for the clarification.